samples/nrf9160/aws_fota/README.rst

```diff
@@ -150,26 +132,12 @@ Before you build the sample, check and update the following configuration option
    Change this value to AWS IoT's MQTT broker.
    To find the address of the AWS IoT MQTT broker, open the AWS IoT console, go to :guilabel:`Test` and select :guilabel:`View endpoint` from the :guilabel:`Connected as XXX` drop-down menu.
 
-.. option:: USE_CLOUD_CLIENT_ID - Custom MQTT client ID
+.. option:: USE_CUSTOM_CLIENT_ID - Custom MQTT client ID
 
    The client ID links your kit to the thing in AWS IoT.
    By default, the client ID is ``nrf-IMEI``, where *IMEI* is the IMEI number of your kit.
```

Comment:
Is this still true? Since you changed it to your_client_id in line 61.

Reply:
That's true I should change line 61 back as I have changed the default again after changing the doc sorry.